### PR TITLE
Add tooltip.css

### PR DESCRIPTION
**Pull request by tiagorangel2011**
Hi! Remember to add tooltip.css to the list of packages.

### CODE REVIEW

File: packages/t/tooltipcss.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "filename": "Tooltips made easy",
+  "description": "A CSS-only tooltip library",
+  "keywords": [
+    "tooltips",
+    "css"
+  ],
+  "license": "Creative Commons - Attribution-NoDerivatives 4.0 International",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tiagorangel2011.git"
```
Comment (MattIPv4) [inline packages/t/tooltipcss.json:12]: This is not a valid git repository

File: packages/t/tooltipcss.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Tooltip.css",
+  "filename": "Tooltips made easy",
+  "description": "A CSS-only tooltip library",
+  "keywords": [
+    "tooltips",
+    "css"
+  ],
+  "license": "Creative Commons - Attribution-NoDerivatives 4.0 International",
```
Comment (MattIPv4) [inline packages/t/tooltipcss.json:9]: This is not a valid SPDX identifier

File: packages/t/tooltipcss.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tiagorangel2011.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/tiagorangel2011.git",
```
Comment (MattIPv4) [inline packages/t/tooltipcss.json:16]: This is not a valid git repository

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: packages/t/tooltipcss.json (hunk 1 above)
Comment (tiagorangel2011) [reply packages/t/tooltipcss.json:12]: Oh, sorry

**Comment by MattIPv4** [comment]
No progress, closing

**Pull request closed without merging**